feature-utils/poly-look/test/react-components/contexts/mocks/poly-import-component-mock.js

```diff
@@ -0,0 +1,24 @@
+import React, { useContext } from "react";
+import { PolyImportContext } from "../../../../src/react-components";
+
+const MockPolyImportComponent = () => {
```

Comment:
This looks more like a fixture than a mock... But I'll check where it's used.

Reply:
what do you mean by fixture?

Reply:
Essentially objects that are created to be the foundation of the rest of the tests. The "Arrange" part in "arrange, act, assert".

Reply:
right right